scripts/path_plans.py

```diff
@@ -260,80 +150,217 @@ def _rotationMatrix_orient_z_axis(self, local_point):
         # Rotation Matrix
         # https://en.wikipedia.org/wiki/Rotation_formalisms_in_three_dimensions#Formalism_alternatives
         rot_matrix = np.stack((vector_x, vector_y, vector_z), axis=1)
-        
+
         return rot_matrix
 
 
+class NewInclinedPlane(DetectedObject):
+    """A container for the incline plane path plan"""
+    def __init__(self, size, locator, orientation, count=(5, 5), clearance=0.25, plane_scale=(2, 2),\
+                 slope=0.5, offset=0.25):
+        # type: (List[float], List[float], np.ndarray, Tuple[int,int], float, Tuple[float,float], float, float) -> None
+        """Initialization function
+
+        @param size:        list[x,y,z] dimensions of presented part
+        @param locator:     center of presented part in world frame coordinates
+        @param orientation: rotation of presented part in world frame coordinates
+        @param count:       Tuple (X,Z) with the count of points along each direction of the plane
```

Comment:
Took me a second to realize what this tuple was composed of in regards to which dimension was effected by which variable, but upon checking documentation here the (X,Z) fixed my concerns. Given the assumption that the concepts of TF's is understood by future developers, I think this is sufficient.

Reply:
You have a good point. I'll think about how this could be clarified by explicitly saying the length and height of the planes.